src/NodaTime.Test/TestHelper.cs

```diff
@@ -496,9 +496,9 @@ internal static void AssertXmlRoundtrip<T>(T value, string expectedXml) where T
         /// </remarks>
         internal static void AssertBinaryRoundtrip<T>(T value)
         {
-            // Can't use [Conditional("!PCL")] as ConditionalAttribute is only positive.
+            // Can't use [Conditional("!NETSTANDARD1_3")] as ConditionalAttribute is only positive.
```

Comment:
Oops, yes. First search and replace was for NETSTANDARD1_3... this one escaped the later refinement.